Hackney.Shared.Processes/Hackney.Shared.Processes.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <PackageId>Hackney.Shared.Processes</PackageId>
+    <RepositoryUrl>https://github.com/LBHackney-IT/processes-shared</RepositoryUrl>
+    <Version>0.0.3</Version>
+    <Authors>London Borough of Hackney</Authors>
+    <Company>London Borough of Hackney</Company>
+    <Copyright>2021</Copyright>
+    <Description>Shared Library for Processes API service</Description>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Hackney.Core.DynamoDb" Version="1.30.0" />
+    <PackageReference Include="Hackney.Core.Http" Version="1.73.0-feat-patch-api-g0004" />
```

Comment:
Why are we using the feature version of this package?

Reply:
Updated to downgrade file.